rio_tiler/tasks.py

```diff
@@ -47,7 +51,10 @@ def create_tasks(
         self, reader: Callable, assets, *args, **kwargs
```

Comment:
can't we do `self, reader: Callable, assets, *args, threads=None, **kwargs` here?